instant/orders-and-signatures.md

```diff
@@ -246,7 +246,21 @@ Finally, package the hashed order with the `EIP-712` domain separator and prefix
 ```python
 encoded_order = keccak(b"\x19Ethereum Signed Message:\n32" + keccak(b"\x19\x01" + DOMAIN_SEPARATOR + hashed_order))
 
-v, r, s = ecdsa_raw_sign(encoded_order, PRIVATE_KEY)
+V, R, S = ecdsa_raw_sign(encoded_order, PRIVATE_KEY)
+
+v = V
+r = Web3.toHex(R)
+s = Web3.toHex(S)
+
+# The ecdsa_raw_sign method we are using may return r & s values that are under 66 characters, so check for
```

Comment:
# bitcoin.ecdsa_raw_sign guarantees r and s <= 66 bytes but not exact and so there is logic to pad '0' to align with bytes32 types

Reply:
do you have a link to the docs by any chance?

Reply:
https://github.com/bitcoin/bips/blob/master/bip-0062.mediawiki#low-s-values-in-signatures consequence of BIP 0062